Client/mods/deathmatch/logic/lua/CLuaTimerManager.cpp

```diff
@@ -108,6 +111,16 @@ void CLuaTimerManager::RemoveAllTimers()
     m_pProcessingTimer = NULL;
 }
 
+void CLuaTimerManager::SetTimerPaused(CLuaTimer* timer, bool paused)
+{
+    assert(timer);
+
+    timer->SetPaused(paused);
+    if (paused)
+        ListRemove(m_ProcessQueue, timer);
```

Comment:
This is not necessary because Lua is single-threaded and `SetTimerPaused` cannot be invoked in the middle of `DoPulse`. At this point `m_ProcessQueue` must be empty.